src/subs/Match.cpp

```diff
@@ -186,18 +186,148 @@ Match::Match(unique_ptr<Detection> e)
           isSolved(false),
           trivialWeights(false) {
     add(std::move(e));
+    // Set null field projection.
+    astrometry::Orientation orient(astrometry::SphericalICRS(0, 0));
+    fieldProjection = new astrometry::Gnomonic(orient);
```

Comment:
```suggestion
    fieldProjection.reset(new astrometry::Gnomonic(orient));
```

Reply:
(to go with the proposed changed to use `unique_ptr` in the header).

It'd be more conventional to move this up into the initializer list, i.e.
```
fieldProjection(new astrometry::Gnomonic(orient))
```
before you open the `{`.